web_services/index.ipynb

```diff
@@ -105,6 +105,163 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding WMS-T(WMS with time support) layers in QGIS"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**How to add WMS-T layers using QGIS** "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. Connect to the Web Map Service in QGIS by following the instructions in [Setting up WMS tutorial](#Tutorial:-Setting-up-WMS).\n",
+    "2. In the QGIS **Browser Panel** navigate to the **WMS/WMTS** location and expand it to show the **DE Africa Services** resource. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_7.png\" alt=\"QGIS - WMS-T layers\" width=\"300\">\n",
+    "\n",
+    "3. Select a WMS-T layer from the **DE Africa Services** resource by either double clicking on the layer in the **Browser Panel** or right clicking on it and selecting **Add Layer to Project** to add the layer onto the map canvas.\n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_8.png\" alt=\"QGIS - WMS-T layers\" width=\"300\">\n",
+    "\n",
+    "4. The name of the loaded WMS-T layer is displayed in the **Layers Panel** with a clock symbol ![temporal indicator](../_static/web_services/mIndicatorTemporal.svg) next it to indicate that it is a temporal layer. \n",
+    "\n",
+    "<img align=\"middle\" src=\"../_static/web_services/ows_tutorial_9.png\" alt=\"QGIS - WMS-T layers\" width=\"500\">\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**How to use the QGIS Temporal Controller with WMS-T layers**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "1. To use the **Temporal Controller** for WMS-T layers, ensure that in the Temporal section of the layer's **Layer Properties**, the **Dynamic Temporal Control** is selected. You can select the time range to be **Automatic** or **Fixed time range**. \n",
```

Comment:
The paragraphs breaks are not working on the final page, can you take a look and fix this.

Reply:
Fixed the paragraph breaks. https://github.com/digitalearthafrica/deafrica-docs/pull/41/commits/60cb4691b9ebec77f87459e5729a0a706dc9fd1c